test/vanilla/acceptanceTests.ts

```diff
@@ -876,12 +875,9 @@ describe('typescript', function () {
         });
       });
 
-      it('should properly handle invalid value for Duration', function (done) {
+      it.skip('should properly handle invalid value for Duration', function (done) {
```

Comment:
We decided against doing validation of duration strings on deserialization, but this test might get re-enabled once we start allowing user preferences to type check during deserialization.